test_lib/compaction.py

```diff
@@ -12,6 +12,7 @@ class CompactionStrategy(Enum):
     SIZE_TIERED = "SizeTieredCompactionStrategy"
     TIME_WINDOW = "TimeWindowCompactionStrategy"
     INCREMENTAL = "IncrementalCompactionStrategy"
+    INMEMORY = "InMemoryCompactionStrategy"
```

Comment:
perhaps renaming to 'IN_MEMORY' would be better.

Reply:
changed